x/auth/vesting/types/vesting_account.go

```diff
@@ -198,54 +196,6 @@ func (bva BaseVestingAccount) MarshalYAML() (interface{}, error) {
 	return marshalYaml(out)
 }
 
-type vestingAccountJSON struct {
```

Comment:
I left a few of these test cases because I thought it would be less likely to cause conflicts if only they were only test cases like this, but well, I'll remove them.